### ISSUE TITLE
Windows 11 install fails

### ISSUE THREAD

**Issue by ksga** [joncage/ed-scout#201]
Did a fresh install of Win11, Chrome, Steam, ED and EDMC.
When trying to install ED-Scout 1.8.2 I'm asked for elevated rights, but a second or so later the install process exits without any messages. Looked in the logbook, but no entries there...
Any suggestions? Are there dependencies I'm not aware of?

**Comment by Tazzetiel** [comment]
The same happens for me on windows 10.

**Comment by ksga** [comment]
Tried downloading the diagnostic build and running it.
The process crashes immediately - but when running as admin things are working as they should.
So maybe something related to system permissions?

**Comment by Kepas-Beleglorn** [comment]
Same here. Setup on Win11 does not work, neither with administrative privileges, nor in compatibility mode.

**Comment by SigmaCeti** [comment]
Same problem. Win10 Home Build 19044.1348

**Comment by eplossl** [comment]
Closing issue #204 in favor of this one.  My machine has the same problem and won't run the installer, either.

**Comment by kwileyk** [comment]
I have the same issue.  Win10 Pro 21H1 19043.1348 - Ryzen 5800X, 32GB RAM, Noctua 3070, in case that's at all relevant.

Based on recommendation from #202, I tried going back a couple versions to see if one of the older ones would work on my computer.

Setup-EdScout-1.7.1.0 worked for me.

Got it from https://github.com/joncage/ed-scout/releases (scroll down to version 1.7.1)

**Comment by mixedd69** [comment]
Same issue for me on W11 latest update. Everything is up to date (drivers, redist, you name it)

Will try @kwileyk suggestion about 1.7.1.0

**Comment by Tazzetiel** [comment]
> i downloaded the https://github.com/joncage/ed-scout/releases/download/v1.8.2/EDScout-DiagnosticBuild.zip, extracted it to a folder, and run it from there, started it before elite dangerous and worked 100% fine

Yeah. This did the trick for me too. It was a bit finicky to get it to actually recognize the nav data (needed like five route replotting and one jump), but by now it seems to work perfectly. Still really annnoyed that the regular "setup version" doesnt seem to work for me (win10)

**Comment by Sakata-MC** [comment]
Did this project die?
No replies to this in a long time despite still being an issue.

**Comment by joncage** [comment]
> Did this project die? No replies to this in a long time despite still being an issue.

Not dead dead but I have two kids 1 and 3 respectively who don't sleep very well so my time and energy for this project has wained somewhat. I do still plan to get back to it (and elite) in the not too distant future though as their sleep patterns _are_ gradually improving.

**Comment by Sakata-MC** [comment]
Congratz, you're nearly to the point you won't be a walking sleep-deprived
zombie again. There's light at the end of the tunnel!
Thanks for the update and take your time! I remember that time in my
kiddo's life well enough -- it was rough.

**Comment by joncage** [comment]
Think this will fix the installer - try v1.9.0 which I'll be pushing up momentarily..

https://github.com/joncage/ed-scout/pull/213
